### ISSUE TITLE
INOP Help please

### ISSUE THREAD

**Issue by AFN40** [Fragtality/PilotsDeck#28]
[StreamDeck0.log](https://github.com/Fragtality/PilotsDeck/files/11139935/StreamDeck0.log)
[StreamDeck1.log](https://github.com/Fragtality/PilotsDeck/files/11139936/StreamDeck1.log)
[FNX2PLD20230403.log](https://github.com/Fragtality/PilotsDeck/files/11139942/FNX2PLD20230403.log)

Hi - I had no issues on the last version I was on. (0.7.7). Now have an issue with INOP across my StreamDeck+ after upgrading FSUIPC to 7.3.19 and the plugin to 0.9, and FenixQuartz to 

I installed the plugin using the installer. I've spent a few hours last night and a few hours this morning looking for the issue but am coming up with nothing. Added my FSUIPC log just in case that's helpful too. I appreciate your help in advance. Thanks! 

[FSUIPC7.log](https://github.com/Fragtality/PilotsDeck/files/11140009/FSUIPC7.log)
[PilotsDeck20230403.log](https://github.com/Fragtality/PilotsDeck/files/11140055/PilotsDeck20230403.log)

**Comment by Fragtality** [comment]
Hmm well, could you maybe Describe the Error you are having? "INOP" tells me exactly nothing other that it is one of the included Images.

Are all Actions in the Error-State? Is it just specific Actions?

**Comment by AFN40** [comment]
Sorry about that. Yes all actions are in error state.

**Comment by Fragtality** [comment]
How and when are you starting all the Tools, Addons and Sim?
Is anything run as admin?

What I find kinda Strange: the Plugin Log starts at 11:27:42 and only 26s later FlightSimulator is started. How did that happen?

**Comment by AFN40** [comment]
That is always running on my computer

**Comment by Fragtality** [comment]
Hmm ... I'm completely puzzled to be honest.
I just can see, that the Sim is started, the Connection-Attempts are working normally and succeed, and then the Plugin is waiting for the Sim to be ready (=that is having clicked ready to fly). All as it should be.
But then the Connection fails all of the sudden.

What happens if you start MSFS normally/manually and don't start any other Addons (except the one which are automatically started through MSFS or FSUIPC)

**Comment by Fragtality** [comment]
Please also try:
Close the StreamDeck Software, start your MSFS as you normally would and begin a flight (= ready to fly was clicked). Then start the StreamDeck Software.
What does happen then?

**Comment by AFN40** [comment]
No luck on either. Here's what I did specifically:
- Emptied out the community folder aside from the necessary items for Fenix & FSUIPC, launched MSFS normally.
- Launched MSFS with community folder filled back up, waited until i saw everything loaded on FSUIPC console, then launched Stream Deck app. 

Last time I had an issue, it was a silly mistake on the FSUIPC7.ini file. Does the below look correct? Anything I can test?

[General]
UpdatedByVersion=7319
InvokeFSUIPCOptionsKey=70,12
AxesWrongRange=No
TCASid=Flight
TCASrange=40,3
AxisCalibration=No
DirectAxesToCalibs=No
AxisIntercepts=No
DontResetAxes=No
ThreadAffinityMask=x0
LuaAffinityMask=x0
LogOptionProtect=Yes
AutoConnectToSim=Yes
AutoExitWithSim=Yes
TimeForLuaClosing=2
TrafficStallTime=2
InitialStallTime=120
NormalStallTime=2
LuaRerunDelay=66
ComReadLoopTime=20
ControlsListBuild=999
Window=-1248,1790,-748,2398
Console=Yes
ConsoleWindowTopMost=No
ConsoleWindow=-2123,-470,-1211,884
PMDG737offsets=Auto
UseSystemTime=No
JoystickTimeout=20
DisconnTrimForAP=No
ZeroElevForAPAlt=No
ThrottleSyncAll=No
EnableExtraButtons=Yes
ShowPMcontrols=No
SpoilerIncrement=512
MagicBattery=No
RudderSpikeRemoval=No
ElevatorSpikeRemoval=No
AileronSpikeRemoval=No
ReversedElevatorTrim=No
ClockSync=No
ClockSyncMins=5
ClearWeatherDynamics=No
PauseAfterCrash=No
BrakeReleaseThreshold=75
SaveDataWithFlights=No
UseProfiles=Yes
WideLuaGlobals=Yes
OpenOnStart=Yes
DebugLua=Yes
NumberOfPumps=0
ComWriteLoopTime=10
FSVersionUsed="KittyHawk",11.0.282174.999
SimConnectUsed=11.0.62651.3

[WideServer]
WideFSenabled=Yes

[WAPI]
EnableWAPI=Yes

[Sounds]
Device1=Primary Sound Driver
Device2=Speakers (7- USB Audio CODEC )
Device3=Speakers (Steam Streaming Speakers)
Device4=Speakers (NVIDIA Broadcast)
Device5=Speakers (Realtek(R) Audio)
Device6=Speakers (Steam Streaming Microphone)
Device7=LG Ultra HD (High Definition Audio Device)
Device8=AORUS FV43U (High Definition Audio Device)
Device9=Speakers (Realtek USB2.0 Audio)
Device10=AORUS FV43U (NVIDIA High Definition Audio)
Device11=LG Ultra HD (NVIDIA High Definition Audio)
Device12=Speakers (2- Realtek(R) Audio)
Device13=LG Ultra HD (2- NVIDIA High Definition Audio)
Device14=Speakers (USB Audio CODEC )
Device15=AORUS FV43U (2- NVIDIA High Definition Audio)

[JoyNames]
0=MFG Crosswind V2/3
0.GUID={BC004220-441C-11ED-8001-444553540000}
1=Bravo Throttle Quadrant
1.GUID={B4B59EB0-DC62-11EC-8002-444553540000}
A=MFG Crosswind V2/3
A.GUID={BC004220-441C-11ED-8001-444553540000}
B=Bravo Throttle Quadrant
B.GUID={B4B59EB0-DC62-11EC-8002-444553540000}
C=T.A320 Pilot
C.GUID={FAC19E80-0E7C-11ED-8001-444553540000}
D=Alpha Flight Controls
D.GUID={1516E350-DC62-11EC-8001-444553540000}
3=T.A320 Pilot
3.GUID={FAC19E80-0E7C-11ED-8001-444553540000}

[JoystickCalibration]
RudderBlendLowest=1

[Axes]
PollInterval=10
RangeRepeatRate=10

[Buttons]
PollInterval=25
ButtonRepeat=20,10

[AutoSave]
Next=1
Interval=60
Files=10
SaveOnGround=No
AutoSaveEnabled=No

[GPSout]
GPSoutEnabled=No

[GPSout2]
GPSoutEnabled=No

[Programs]
RunIf1=READY,KILL,C:\PilotsDeck_FNX2PLD\PilotsDeck_FNX2PLD.exe

[LuaFiles]
1=FNX320_AUTO
2=FNX320_SYNC
3=GSX_AUTO
4=GSX_AUTO_FNX
5=PilotsDeck
6=747_8
7=DSA_GSX_AUTO
8=PilotsDeck_737

[Auto]
1=Lua GSX_AUTO
2=Lua FNX320_AUTO
3=Lua FNX320_SYNC

[Profile.FNX320]
1=FNX320

**Comment by Fragtality** [comment]
That was not what I meant ^^

You don't need to empty out the Community Folder. If you also moved away the MobiFlight Module at that Instance it is clear that the Plugin won't work ;)

Don't know what Situation "everything loaded in the FSUIPC Console" should be. Like I said: start a Session, click ready to fly, and when you're in the Plane sitting at the Gate, start the StreamDeck SW again.

If the ini made Problems before, you can always move it away in a safe place and let fsuipc generate a new one and see if that helps. If it does not you can copy over your old ini again.

**Comment by AFN40** [comment]
Sorry for the misunderstanding. 

But I have good news and I have bad news. Good news is of course I think i got everything operational. Bad news, is I don't know what the true fix was. 

I removed the ini file, relaunched the sim. Then quit out. Added back in the GSX/FNX320 items to the ini file, relaunched the sim and it works. 

Appreciate you being so responsive. I'll keep you posted after my first full flight with it back in working order to see if there's anything off.
